### ISSUE TITLE
Multiple OIDs in Get and GetBulk

### ISSUE THREAD

**Issue by thugcee** [ptek/netsnmp#7]
Sorry if this is a stupid question but I'm just starting adventure with Haskell and don't understand its code well.
`netsnmp` is the 3rd Haskell SNMP library I'm checking out and again, looking through the code, it looks like it doesn't support getting multiple vars (OIDs) in a single Get request. Is it true?

**Comment by ptek** [comment]
Hi, its not stupid at all. I have to admit that I have not been using the package for a long time already myself. 
You can use snmpBulkWalk - which starts with an OID and returns the subtree of the values.
Snmp Get is designed to get only one value at a time as I remember.

**Comment by thugcee** [comment]
Net-SNMP `snmpBulkWalk` function, which is implemented using `SNMP GetBulk` request, allows requesting for one OID "column" (not useful for me), but SNMP's `Get` (only non-repeating OIDs) and `GetBulk` (non-repeating and repeating OIDs) allow putting multiple OIDs in one request. This can improve performance significantly.

**Comment by ptek** [comment]
You are right. In fact, the code uses `getBulk` to do the `snmpBulkWalk`. Maybe you could fork the code and refactor it to expose `snmpBulkGet` with a nice interface?

**Comment by ptek** [comment]
@thugcee I know its 4 years later, but if you have implemented the `snmpBulkGet`, I would happily incorporate it into the library.

**Comment by thugcee** [comment]
No, sorry, I haven't implemented it. Unfortunately I had to change tech stack completely for other reasons.
